comparison_pipeline/transform.py

```diff
@@ -17,12 +17,13 @@ def format_authors(authors: str) -> list[str]:
     """Returns a list of authors if they exist and ['No Author'] if none detected"""
 
     if authors.lower() == "nan":
-        return ["No Author"]
+        return None
```

Comment:
Will this work? Is the entry for "nothing there" actually spelled out "nan"?

Reply:
Tried and tested locally